### PR TITLE
docs/s3-compatibility.md: add detail for GetBucketCors

### PR DESCRIPTION
**Pull request by ferristocrat**
Adds documentation for GetBucketCors method

## PR Checklist

- [x] The PR title is formatted as follows: `miniogw: add TestXYZ for XYZ`
  - [x] The package name goes before the colon
  - [x] The part after the colon uses the verb tense + phrase that completes the blank in, "This change modifies Gateway-ST to ___________"
  - [x] Lowercase verb after the colon
  - [x] No trailing period
  - [x] Keep the title as short as possible. Ideally under 72 characters or shorter
  - [x] No Markdown

## Commit Message Body

- What, why, please describe the tests and the performance impact (if any)
- No Markdown
- The body is wrapped at 72 characters unless it's really needed (ASCII art, table, or long link)
- If there is a corresponding issue, add either `Closes #1234`, `Fixes #1234`, `Resolves #1234` or `Updates #1234` (the latter if this is not a complete fix) to this comment
- If referring to a repo other than `storj/gateway-st`, you can use the `owner/repo#issue_number` syntax: `Fixes storj/common#1234`
- We do not use Signed-off-by lines. Please don't add them. Our Gerrit server & GitHub bots enforce CLA compliance instead
- More: https://github.com/storj/docs/blob/main/code/Git.md
- Delete these instructions once you have read and applied them

## Code Review Checklist (to be filled out by reviewer)

 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs, it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation needs updating?
 - [ ] Do the database access patterns make sense?
 - [ ] Copy the Commit Message Body section contents into the submit prompt upon PR completion

### CODE REVIEW

**Review by egonelbre — APPROVED** [review]

File: docs/s3-compatibility.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ S3 Compatibility
 | GetBucketAccelerateConfiguration            | No      | No                                                           |                                                     |
 | GetBucketAcl                                | No      | No                                                           |                                                     |
 | GetBucketAnalyticsConfiguration             | No      | No                                                           |                                                     |
-| GetBucketCors                               | No      | No                                                           |                                                     |
+| GetBucketCors                               | Partial      | No                                                           |    See GetBucketCors section below                                                 |
```
Comment (halkyon) [inline docs/s3-compatibility.md:33]: nit: the indentation doesn't line up with the rows above and below

File: docs/s3-compatibility.md [hunk 2]
```diff
@@ -167,6 +167,26 @@ improve object consistency in
 cloud](https://www.storj.io/blog/secure-access-control-in-the-decentralized-cloud)
 is a good read for why we don't support ACL-related actions.
 
+### GetBucketCors
+While the `GetBucketCors` method is supported, `PutBucketCors` and `DeleteBucketCors` are not available. The response body returned when making a `GetBucketCors` request will always be the following:
```
Comment (amwolff) [inline docs/s3-compatibility.md:171]:
```suggestion
### GetBucketCors

While the `GetBucketCors` method is supported, `PutBucketCors` and
`DeleteBucketCors` are not available. The response body returned when
making a `GetBucketCors` request will always be the following:
```

File: docs/s3-compatibility.md (hunk 1 above)
Comment (amwolff) [inline docs/s3-compatibility.md:33]:
```suggestion
| GetBucketCors                          | Partial      | No                                                           |    See GetBucketCors section below                                                 |
```

**Review by amwolff — CHANGES_REQUESTED** [review]

**Comment by amwolff** [comment]
@ferristocrat are you still working on this?

**Comment by amwolff** [comment]
S3 Compatibility is now hosted at https://docs.storj.io/dcs/api/s3/s3-compatibility.

**Pull request closed without merging**